lib/eventboss/error_handlers/rollbar.rb

```diff
@@ -0,0 +1,11 @@
+module Eventboss
+  module ErrorHandlers
+    class Rollbar
+      def call(exception, context = {})
```

Comment:
@tomaszkrzywda just received Dependabot notification about this change 👏 

What do you think about sending `context[:processor].jid` to Rollbar as well? We use `jid` to match errors with Kibana logs:

https://github.com/AirHelp/ahplus/blob/8216b5357797e5ee9d193ead58be9870f7ff706e/config/initializers/eventboss.rb#L5

Reply:
@wkosak I can't think of any reason not to do it:). Rollbar.error accepts additional data as well.